.github/workflows/playwright_asset_exchange.yml

```diff
@@ -1,9 +1,11 @@
 name: Playwright Tests - Asset Exchange
+
 on:
-  pull_request:
-  push:
-    branches:
-      - main
+  workflow_dispatch: {}
+  workflow_run:
+    workflows: ["Prepare Dependencies"]
+    types:
+      - completed
```

Comment:
even though I have added this, the workflow didn't run after the `Prepare Dependencies` was successfull because this workflow change doesn't exist on  `main` branch, and it's only considered when it's on `main`.